assets/files.wxs

```diff
@@ -1628,6 +1628,15 @@
         <Component Id="cmp70A5A090C6B94E8F8FC9B4AE5EC2969B" Guid="{5cf8209b-8488-4893-a0f3-980efa86fdf3}">
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Security.Principal.dll; SideIndicator=<=}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 120
120:         $diffs | Should -BeExactly $null
```